src/exec.rs

```diff
@@ -97,6 +99,7 @@ struct ExecReadOnly {
 /// Facilitates the construction of an executor by exposing various knobs
 /// to control how a regex is executed and what kinds of resources it's
 /// permitted to use.
+#[derive(Debug)]
```

Comment:
Is this `Debug` impl necessary?

Reply:
`ExecBuilder` is public via the `internal` module which "exists to support suspicious activity".

I'll remove the debug impl and allow the lint.